### PR TITLE
Created a new document to help guide people through the Pull Request process

### PR DESCRIPTION
**Pull request by joshuamil**
Created new PULL-REQUESTS.md file
Updated link in the README.md to include this link

Please provide feedback on both the formatting and the content of this document. There are lots of things that we can review during the PR process, so please feel free to suggest changes, or submit your own PR's to expand on this guide.

### CODE REVIEW

File: PULL-REQUESTS.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+1. Remove all commented-out / hidden code unless including an explanation as to why it is being temporarily hidden
+1. If you have copied/pasted code from an online resource, read and review it yourself first. Ensure it meets your standards / requirements.
+1. Make sure there are no unnecessary files changed
+1. Limit your PR's to a single story and/or ticket
+1. Limit the size of your PR's as much as possible
+
+## When Submitting a PR
+Aside from writing good, clean code, completing the tasks you're assigned and then running through a "pre-submit checklist", there are some guidelines for how to submit a Pull Request to ensure things go as smoothly as possible:
+
+1. Make sure to highlight dependencies on other branches/projects/versions (e.g.: requires API v2, requires API branch `foo` be merged)
```
Comment (kaityhallman) [inline PULL-REQUESTS.md:34]: Perhaps also add if once the branch gets merged in, if a `yarn install` or `npm install` is necessary

File: PULL-REQUESTS.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+1. Expect feedback / changes to come; be open to and receptive of the feedback
+1. Immediately review your own PR as if you were the reviewer. Use the checklist below to check your own work before you ask for peer review.
+
+## When Reviewing a PR
+Reviewing Pull Requests can be a time consuming process, but it's your opportunity to ensure quality, enforce standards and understand the application you're working on at a deeper level. Set aside time to review code, do it consistently and properly and treat it as important work. There are dozens of things to look for when reviewing code and this is by no means an exhaustive list, but rather a basic guideline to follow for starting the review process:
+
+### General Guidelines
+1. Leave positive, helpful feedback
+1. Be respectful: Avoid derogatory or condescending words and/or tone
+1. Check-out the code and run it in your environment
```
Comment (kaityhallman) [inline PULL-REQUESTS.md:51]: Also checking for console errors!

File: PULL-REQUESTS.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+## Beginning the Work
+Good Pull Requests start well before you write your first line of code. Be prepared to craft discreet, focused and intentioned PR's by following a few simple guidelines:
+
+1. Create a __new__ branch
+1. Select a branch name that matches your company's guidelines
+1. In the absence of guidelines, use a consistent, meaningful branch naming convention. Consider using the story or task identifier in the branch name.
+1. Keep the work to a single feature / story / task to limit the scope of the upcoming PR
+1. Use meaningful commit messages:
+  * Great: `ABC-123: Updated logic to include the correct coefficient multiplier based on the information provided by @teammate in our grooming session on 10/02`
```
Comment (kirillian) [inline PULL-REQUESTS.md]: I would consider following the 80 character limit when possible for the first line of your commit. You can put longer messages on the next couple. Maybe something like this:
```
ABC-123: Updated logic to include the correct coefficient multiplier

- based on the information provided by @teammate in our grooming session on 10/02
```

File: PULL-REQUESTS.md (hunk 2 above)
Comment (joshuamil) [reply PULL-REQUESTS.md:51]: Good catch! Updated accordingly.

File: PULL-REQUESTS.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+1. Make sure all Linter warnings and errors have been addressed
+1. If you don't have a Linter, ensure your indentation, spacing and formatting is correct and consistent
+1. Make sure you have properly commented your code. If your team uses JSDoc, JavaDoc or something similar, make sure you're compliant
+1. Remove all superfluous logging statements (e.g.: `console.log`, `console.warn`, `System.out.println`)
+1. Remove all commented-out / hidden code unless including an explanation as to why it is being temporarily hidden
+1. If you have copied/pasted code from an online resource, read and review it yourself first. Ensure it meets your standards / requirements.
+1. Make sure there are no unnecessary files changed
+1. Limit your PR's to a single story and/or ticket
+1. Limit the size of your PR's as much as possible
+
```
Comment (kirillian) [inline PULL-REQUESTS.md:30]: Maybe add something like:
```
+1. If you had to make some hard decisions in your code, consider preemptively explaining these to the potential reviewers to head off questions from them and give yourself the chance to explain yourself without being on the back foot.
```

File: PULL-REQUESTS.md (hunk 1 above)
Comment (joshuamil) [reply PULL-REQUESTS.md:34]: This is a great one, it seems to bite me a couple of times a week!

File: PULL-REQUESTS.md (hunk 4 above)
Comment (kirillian) [reply PULL-REQUESTS.md:30]: This appears to be similarish to what you wrote below, so with 'Add inline comments in the PR for anything you feel might warrant explanation'. Maybe adding additional information to that line would be preferable?

Comment (joshuamil) [reply PULL-REQUESTS.md:30]: I like that @kirillian, the comment about "hard decisions" is a good note and I like emphasizing preemptive commentary. I'll make this change.

File: PULL-REQUESTS.md (hunk 3 above)
Comment (joshuamil) [reply PULL-REQUESTS.md]: Yeah, that's a good catch. I didn't realize I'd hit 80 chars in that line.

Comment (kirillian) [reply PULL-REQUESTS.md]: I figured if we're going to be giving them advice, we should not uh give them too terrible of advice in some other manner. :)

**Pull request merged**